server_admin/topics/clients/oidc/proc-using-a-service-account.adoc

```diff
@@ -0,0 +1,65 @@
+[id="proc-using-a-service-account_{context}"]
+[[_service_accounts]]
+
+==== Using a Service Account
+
+Each OIDC client has a built-in _service account_ which allows it to obtain an access token.
+
+.Prerequisites
```

Comment:
We need a comma after _service account_ because it is followed by "which" not "that."